hack/local-up-cluster.sh

```diff
@@ -362,13 +390,23 @@ function start_apiserver {
     # Ensure CERT_DIR is created for auto-generated crt/key and kubeconfig
     sudo mkdir -p "${CERT_DIR}"
 
+    # Create client ca
+    sudo /bin/bash -e <<EOF
+    rm -f "${CERT_DIR}/client-ca.crt" "${CERT_DIR}/client-ca.key"
+    openssl req -x509 -sha256 -new -nodes -days 365 -newkey rsa:2048 -keyout "${CERT_DIR}/client-ca.key" -out "${CERT_DIR}/client-ca.crt" -subj "/C=xx/ST=x/L=x/O=x/OU=x/CN=ca/emailAddress=x/"
+    echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","client auth"]}}}' > "${CERT_DIR}/client-ca-config.json"
+EOF
+    create_client_certkey kubelet
+    create_client_certkey kube-proxy
```

Comment:
I'd like to see this as another `system:kube-proxy` style user.   We've reserved the `system:`  prefix throughout the authn/authz layers for "the kube cluster relies on this stuff, don't muck with it unless you really know what you're doing" resources.